cmd/cosign/cli/verify/verify_blob_test.go

```diff
@@ -255,24 +250,28 @@ func TestVerifyBlob(t *testing.T) {
 			name:         "valid signature with public key",
 			blob:         blobBytes,
 			signature:    blobSignature,
-			sigVerifier:  signer,
+			key:          pubKeyBytes,
 			experimental: false,
 			shouldErr:    false,
 		},
+		/* TODO:
+		This currently passes without error because we don't require an experimental
+		lookup for public keys. Add this test back in when we have a --verify-tlog-online
 		{
 			name:         "valid signature with public key - experimental no rekor fail",
 			blob:         blobBytes,
 			signature:    blobSignature,
-			sigVerifier:  signer,
+			key:          pubKeyBytes,
```

Comment:
looking through the code, i am unsure why this isn't erroring out. in verifyInternal, it seems like we should be falling back to an online lookup if bundle verification fails

Reply:
Never mind, dug into the code more, I see that it doesn't err out, just returns that the bundle wasn't verified.